### PR TITLE
Updated inputfields of blog

### PR DESCRIPTION
**Pull request by koenhamstra**
Updated the value of the inputfields. Except for subgoals.

### CODE REVIEW

File: database/seeders/ProgramSeeder.php [hunk 1]
```diff
@@ -15,13 +15,172 @@ class ProgramSeeder extends Seeder
...
+        ]);
+        DB::table('programs')->insert([
+            'name' => 'Schakelcursus Wiskunde',
+        ]);
+        DB::table('programs')->insert([
+            'name' => 'Seksualiteit en seksueel misbruik',
         ]);
         DB::table('programs')->insert([
             'name' => 'Social Work',
         ]);
```
Comment (AnnevanHeeren) [inline database/seeders/ProgramSeeder.php:142]: Uhh what happened here

**Review by avanheeren — APPROVED** [review]

**Review by awqfrxx — APPROVED** [review]

**Comment by AlexGit123** [comment]
conflicts with the DB seeder

File: resources/views/blogs/create.blade.php [hunk 2]
```diff
@@ -15,14 +15,15 @@
         @csrf
         <div class="field is-horizontal">
             <div class="field-label is-normal">
-                <label class="label">Program</label>
+                <label class="label">Programs</label>
```
Comment (Koreyoshi159) [inline resources/views/blogs/create.blade.php:18]: why change it from singular to plural?

**Review by AlexGit123 — CHANGES_REQUESTED** [review]
conflicts with db seeder

File: resources/views/blogs/create.blade.php (hunk 2 above)
Comment (awqfrxx) [reply resources/views/blogs/create.blade.php:18]: inconsistencies with the rest, we had singular and plural mixed between every input field

File: resources/views/blogs/create.blade.php [hunk 3]
```diff
@@ -38,14 +39,15 @@
...
-                <label class="label">Type of Activity</label>
+                <label class="label">Type of Activities</label>
             </div>
             <div class="field-body">
                 <div class="field">
                     <div class="control">
                         <div class="select is-fullwidth @error('activity_id') is-danger @enderror">
                             <select class="select-activity" type="text"
                                     id="activity_id" name="activity_id">
+                                <option disabled selected value=""> -- Select a Activity --</option>
```
Comment (avanheeren) [inline resources/views/blogs/create.blade.php]: an activity*

File: resources/views/blogs/create.blade.php [hunk 4]
```diff
@@ -87,18 +90,19 @@
...
+                            <select class="select-policy" type="text" id="business_operation_id"
+                                    name="business_operation_id">
+                                <option disabled selected value=""> -- select a Business Operation --</option>
                                 <option selected value=""> None</option>
-                                @foreach ($sdgs as $sdg)
-                                    <option value="{{$sdg->id}}" {{ (old('sdg') == $sdg->id ? "selected":"") }}>
-                                        {{ $sdg->id }}. {{ $sdg->name }}
+                                @foreach ($business_operations as $business_operation)
+                                    <option value="{{ $business_operation->id }}" {{ (old('business_operation') == $business_operation->id ? "selected":"") }}>
+                                        {{ $business_operation   ->name }}
```
Comment (avanheeren) [inline resources/views/blogs/create.blade.php]: why space after operation  :'(

**Review by avanheeren — CHANGES_REQUESTED** [review]
requesting changes

**Review by Sanyi0001 — APPROVED** [review]

File: resources/views/blogs/create.blade.php (hunk 4 above)
Comment (koenhamstra) [reply resources/views/blogs/create.blade.php]: fixed it

**Review by Koreyoshi159 — APPROVED** [review]
nice

**Pull request merged**